src/Development/Rattle/Hash.hs

```diff
@@ -24,65 +25,99 @@ import System.FilePath
 import Numeric
 import Control.Exception.Extra
 import Control.DeepSeq
+import GHC.Generics
+import Data.Serialize
+import General.FileName
+import General.FileInfo
 
+newtype Hash = Hash BS.ByteString
```

Comment:
Would be good to get a comment. Is this a series of bytes? Or a series of Hex values? What is the expected length? Note that it used to be a series of Hex values, but I suspect now its a byte string a series of bytes would be way more appropriate (not sure if you made that change).

Reply:
I'm not sure about this one. I definitely made this change out of necessity, but I'm not sure of the answers to your questions.  The Bytestring is viewed as containing 8 bit characters (according to bytestring docs).